src/main/groovy/org/sba/excel/poi/reader/ExcelWorkSheetHandler.groovy

```diff
@@ -144,7 +137,7 @@ class ExcelWorkSheetHandler implements SheetContentsHandler, ExecutionContextAwa
      */
     @Override
     void endRow(int rowNum) {
-        if (!shouldSkipRow() && rowNum <= HEADER_ROW) {
+        if (!shouldSkipRow() && rowNum <= headerRow) {
```

Comment:
It means we skip every row that's "above" the header row. We could be less restrictive on that one.

moreover, if we decide to keep that condition, we should probably add a special handler to let the client react

Reply:
i didn't want to change this code but you're rigth we can ignore row after header row

i change it